packages/lib/src/types/task.ts

```diff
@@ -48,6 +48,29 @@ export enum TaskStatus {
   KnownIssue = "known-issue",
 }
 
+// Task status sorted in order of "relevance", with the most likely actionable at the top.
+export const SortedTaskStatus = [
```

Comment:
Cool, I removed them from the legend as well in this case since I think we probably should include them everywhere or nowhere. Let me know if you disagree!